static/app/components/profiling/functionsMiniGrid.tsx

```diff
@@ -44,25 +55,33 @@ export function FunctionsMiniGrid(props: FunctionsMiniGridProps) {
 
       {functions &&
         functions.map((f, idx) => {
-          const exampleProfileIdRaw = f.worst;
+          if (!defined(f)) {
+            return null;
+          }
+
+          const exampleProfileIdRaw = f['examples()']![0];
           const exampleProfileId = exampleProfileIdRaw.replaceAll('-', '');
           return (
             <Fragment key={idx}>
-              <FunctionsMiniGridCell title={f.name}>
+              <FunctionsMiniGridCell title={f.function as string}>
```

Comment:
Lots of type casts in this file component, are the types wonky?

Reply:
because we can now specify which columns we want to return, the return type on the results is `{[key: string]: string[] | string | number | null}`. We can probably do better and avoid the type casts but I didn't want to make this PR any larger than it already is. Something to improve upon for sure